parser/prism/Parser.h

```diff
@@ -11,6 +11,7 @@ extern "C" {
 
 #include "core/LocOffsets.h"
 #include "parser/Node.h" // To clarify: these are Sorbet Parser nodes, not Prism ones.
+#include "parser/parser.h"
```

Comment:
Can you please extract `ParseResult` into its own `parser/ParseResult.h`, so we don't have to pull in the entire `parser/parser.h`?

See: https://github.com/Shopify/sorbet/pull/618/files#diff-e2e9659ca78314a30b51dae3f39603cf8a8cff07f9fd47b9f40c5fd1967151a9

```suggestion
#include "parser/ParseResult.h"
```

Reply:
Done in a new commit.